cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -334,4 +335,15 @@ describe('events dashboard', () => {
       expect(response.body.errors).not.to.exist;
     });
   });
+
+  it('chapter admin should be allowed to send email to attendees', () => {
+    const eventId = 1;
+
+    // admin of chapter 1
+    cy.login('admin@of.chapter.one');
+
+    cy.sendEventInvite(eventId, ['confirmed']).then((response) => {
+      expect(response.body.errors).not.to.exist;
+    });
+  });
```

Comment:
Ideally these simple api calls should be to check that users who don't have permission can't do the thing.  

Could you try changing the `cy.login` in `beforeEach` to use `admin@of.chapter.one`? Then this 'it' can be used to check that `admin@of.chapter.two` and `banned@chapter.admin` can't.